build.sbt

```diff
@@ -23,13 +24,43 @@ lazy val root = (project in file("."))
       )
     },
     scriptedBufferLog := false,
+    scriptedSbt := "1.10.2",
```

Comment:
Technically speaking we could find the lowest sbt version that this plugin can support and set `scriptedSbt` to this version but honestly I don't think its needed, anyone who will use this plugin will likely be running latest version of sbt anyways and we can always fix this later if needed